### PR TITLE
Improving the distributed deployment overview

### PR DESCRIPTION
**Pull request by nilminiwso2**
Refactoring the Distributed Deployment overview page.

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 1]
```diff
@@ -1,361 +1,104 @@
...
-            Developer Portal
-        </td>
-        <td>
-            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
+            Only starts the Traffic Manager component.
         </td>
     </tr>
 </table>
 
-## Databases for API-M components
```
Comment (rmsamitha) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:62]: Sectionn on "Databases required for each profiles" should not be dropped.

File: en/docs/install-and-setup/setup/reference/default-product-ports.md [hunk 2]
```diff
@@ -12,14 +12,144 @@ This page describes the default ports used by each runtime of WSO2 API Manager.
...
+		<td>
+			<code>5672</code>
+		</td>
+		<td>
+			Port of the internal Message Broker of the API-M runtime.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>7611</code>
```
Comment (isharac) [inline en/docs/install-and-setup/setup/reference/default-product-ports.md:58]: The thrift port is not in use now AFAIK. @tharindu1st Please confirm.

**Review by isharac — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 1 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:62]: Ack. 
I moved it to the 'Databases' section. Will keep it under 'Distributed Deployment' and link back and forth between the database doc and distributed deployment doc.

Comment (rmsamitha) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:62]: Noticed that is moved to "en/docs/install-and-setup/setup/setting-up-databases/overview.md".  Is there a specific reason to move? Knowledge on dbs that each node needs should better to be at here AFAIU.

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 3]
```diff
@@ -1,361 +1,104 @@
...
-            Analytics dashboard and business rule portals
+            Enables consumers to self-register, discover API functionality, subscribe to APIs, evaluate them, and interact with API publishers.
         </td>
     </tr>
-    </tbody>
-    </table>
--->
\ No newline at end of file
+</table>
+
```
Comment (rmsamitha) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:100]: Don't we add a diagram is this page on distributed setup with all the conections as in https://apim.docs.wso2.com/en/3.2.0/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m/ ?

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 4]
```diff
@@ -1,361 +1,104 @@
...
-!!! Note
-    Work in progress!
-
-When we consider a distributed deployment of WSO2 API Manager, we have the option of separating the five components and clustering each component as needed. Let's look more closely at how the API Manager components are deployed separately.
-
-In the following diagram, the five components are set up in a distributed deployment, and the five databases are connected to the relevant components respectively. The entire setup is also fronted by a load balancer.
-
-[![Understanding the distributed deployment]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)]({{base_path}}/assets/img/setup-and-install/db-connections-distributed-deployment.png)
-
-??? "Click here for more details on default communication ports among components."
```
Comment (rmsamitha) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:126]: Why do we drop this sectio "Click here for more details on default communication ports among components." ?

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 5]
```diff
@@ -1,361 +1,104 @@
...
+            Only starts the Traffic Manager component.
         </td>
     </tr>
 </table>
 
-## Databases for API-M components
+## API-M Components
 
-WSO2 API Manager components use the databases given below. Find out more about [databases in API-M]({{base_path}}/install-and-setup/setup/setting-up-databases/overview).
+Listed below are the five components of the API-M runtime and its function.
```
Comment (rmsamitha) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:49]: Add a statement similar to below here to specify the relationship of the profiles with componets, in a distributed setup. 
"When you run API-M instances in the recommended profiles, below components will be available in the rutime instances appropriately, that will operate the related functionalities as expected by each profile."

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 3 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:100]: @isharac

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 4 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:126]: @isharac - Can we get this content for API-M 4 as well.. The previous content was outdated for API-M, hence I removed them.

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 1 above)
Comment (rmsamitha) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:62]: > Ack.
> I moved it to the 'Databases' section. Will keep it under 'Distributed Deployment' and link back and forth between the database doc and distributed deployment doc.

Ack

Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:62]: I thought.. we can link back from the 'Distributed Setup Overview" to the Databases section. However, I missed adding that link.. 
I think it is okay to do what you suggested and have links going both ways..

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 6]
```diff
@@ -1,361 +1,104 @@
...
```
[the remark this answers is not in the document]
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md]: Changed all the relevant instances of 'runtime' to 'server'.

File: en/docs/install-and-setup/setup/reference/default-product-ports.md (hunk 2 above)
Comment (tharindu1st) [reply en/docs/install-and-setup/setup/reference/default-product-ports.md:58]: yes 7611 and 7711 we can remove from docs

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 3 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:140]: Will send a separate PR for this one when the diagrams are ready.

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md (hunk 4 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:126]: Will send a separate PR for this one when the diagrams are ready.

File: en/docs/install-and-setup/setup/reference/default-product-ports.md (hunk 2 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/reference/default-product-ports.md]: fixed

File: en/docs/install-and-setup/setup/setting-up-databases/overview.md [hunk 7]
```diff
@@ -32,8 +34,9 @@ WSO2 products are shipped with scripts for creating the required tables in all t
 
 !!! Note
 
-    - To enable the default H2 database, add the below configurations to deployment.toml file in <APIM_HOME>/repository/conf and restart the server.
-          ```toml
-          [database_configuration]
-          enable_h2_console = true
-          ```
\ No newline at end of file
+    To enable the default H2 database, add the below configurations to deployment.toml file in <APIM_HOME>/repository/conf and restart the server.
```
Comment (rmsamitha) [inline en/docs/install-and-setup/setup/setting-up-databases/overview.md:37]: This has been incorrect from earlier. Shall we fix it too from this PR. It should be fixed as below.
"To eable browsing the H2 database from a web browser, add the below configurations to deployment.toml..."

**Review by rmsamitha — APPROVED** [review]

**Review by isharac — APPROVED** [review]

File: en/docs/install-and-setup/setup/setting-up-databases/overview.md (hunk 7 above)
Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/setting-up-databases/overview.md]: fixed.

**Pull request merged**